.github/codecov.yml

```diff
@@ -13,20 +13,22 @@
 # limitations under the License.
 
 codecov:
-    strict_yaml_branch: "master"
```

Comment:
I'm assuming you removed `strict_yaml_branch: "master"` for test purposes, but it should be added back before merging.

Reply:
sure, will change it back. btw, why it need to be strict to master?

Reply:
This tells codecov to only use the configuration that is in `master` -- otherwise, every PR could overwrite its own code coverage settings.